src/app/loadout-builder/filter/ExoticPicker.tsx

```diff
@@ -142,15 +136,19 @@ function filterAndGroupExotics(
 
 /** A drawer to select an exotic for your build. */
 export default function ExoticPicker({ lockedExoticHash, classType, onSelected, onClose }: Props) {
+  classType: DestinyClass;
   const defs = useD2Definitions()!;
   const language = useSelector(languageSelector);
   const [query, setQuery] = useState('');
 
   const allItems = useSelector(allItemsSelector);
+  const vendorItems = useSelector((state: RootState) =>
+    includeVendorItems ? characterVendorItemsSelector(state, storeId) : emptyArray<DimItem>(),
```

Comment:
LO itself uses a `loVendorItemsSelector` that wraps `characterVendorItemsSelector` in an explicit vendor allow-list - it'd be good if both of these used the same source of truth so that we don't end up showing a dummy exotic in the exotic picker that LO won't end up considering.